spec/features/finance/payment_breakdown_spec.rb

```diff
@@ -44,12 +74,255 @@ def and_i_click_on(string)
+        params: {
+          participant_id: participant.user.id,
+          course_identifier: "ecf-induction",
+          declaration_date: (participant.schedule.milestones.second.start_date + 1.day).rfc3339,
+          created_at: participant.schedule.milestones.second.start_date + 1.day,
+          cpd_lead_provider: lead_provider.cpd_lead_provider,
+          declaration_type: RecordDeclarations::ECF::RETAINED_ONE,
+          evidence_held: "other",
+        },
+      )
+      started_declaration = ParticipantDeclaration.find(JSON.parse(serialized_started_declaration).dig("data", "id"))
+      started_declaration.make_eligible!
+      started_declaration.make_payable!
+      started_declaration.update!(
+        statement: nov_statement,
+      )
+    end
+  end
+
+  def create_retained_declarations_jan_mentor(participant)
+    timestamp = participant.schedule.milestones.second.start_date + 1.day
+    travel_to(timestamp) do
+      serialized_started_declaration = RecordDeclarations::Retained::Mentor.call(
+        params: {
+          participant_id: participant.user.id,
+          course_identifier: "ecf-mentor",
+          declaration_date: (participant.schedule.milestones.second.start_date + 1.day).rfc3339,
+          created_at: participant.schedule.milestones.second.start_date + 1.day,
+          cpd_lead_provider: lead_provider.cpd_lead_provider,
+          declaration_type: RecordDeclarations::ECF::RETAINED_ONE,
+          evidence_held: "other",
+        },
+      )
+      retained_declaration = ParticipantDeclaration.find(JSON.parse(serialized_started_declaration).dig("data", "id"))
+      retained_declaration.make_eligible!
+      retained_declaration.make_payable!
+      retained_declaration.update!(
+        statement: jan_statement,
+      )
+    end
+  end
+
+  def create_retained_declarations_jan_ect(participant)
+    timestamp = participant.schedule.milestones.second.start_date + 1.day
+    travel_to(timestamp) do
+      serialized_started_declaration = RecordDeclarations::Retained::EarlyCareerTeacher.call(
+        params: {
+          participant_id: participant.user.id,
+          course_identifier: "ecf-induction",
+          declaration_date: (participant.schedule.milestones.second.start_date + 1.day).rfc3339,
+          created_at: participant.schedule.milestones.second.start_date + 1.day,
+          cpd_lead_provider: lead_provider.cpd_lead_provider,
+          declaration_type: RecordDeclarations::ECF::RETAINED_ONE,
+          evidence_held: "other",
+        },
+      )
+      retained_declaration = ParticipantDeclaration.find(JSON.parse(serialized_started_declaration).dig("data", "id"))
+      retained_declaration.make_eligible!
+      retained_declaration.update!(
+        statement: jan_statement,
+      )
+    end
+  end
+
+  def nov_retained_breakdowns_are_calculated
+    @nov_retained_1 = Finance::ECF::CalculationOrchestrator.new(
```

Comment:
it would be preferable to stick to the classic to RSpec `let` at the top of the file, at least for consistency. Also these can be mutated at any point in time during the test run and make things hard to debug when this happens. you  have to jump up and down the file when trying to see where this was instantiated, then mutated, then go back to check the expectation, making keeping track of what's happening in the spec rather difficult.